### PR TITLE
Fix ChromeDriver acceptInsecureCerts capability tests

### PR DESCRIPTION
**Pull request by JohnChen0**
PR #19402 added acceptInsecureCerts capability to Chrome configuration
by default. This causes failures in tests that explicitly check this
capability. Fixing by removing the default acceptInsecureCerts
capability during new_session tests.

### CODE REVIEW

**Comment by JohnChen0** [comment]
@Hexcles PTAL

File: webdriver/tests/new_session/conftest.py [hunk 1]
```diff
@@ -25,6 +25,19 @@ def add_browser_capabilities(capabilities):
     return add_browser_capabilities
 
 
+@pytest.fixture(name="configuration")
+def fixture_configuration(configuration):
+  """Remove "acceptInsecureCerts" from capabilities if it exists.
+
+  Some browser configurations add acceptInsecureCerts capability by default.
+  Remote it during new_session tests to avoid interference.
```
Comment (Hexcles) [inline webdriver/tests/new_session/conftest.py]:
```suggestion
  Remove it during new_session tests to avoid interference.
```

**Review by Hexcles — COMMENTED** [review]
Looks like Firefox always adds `acceptInsecureCerts`. I'm wondering why the problem only arose after #19402 .

**Comment by JohnChen0** [comment]
> Looks like Firefox always adds `acceptInsecureCerts`. I'm wondering why the problem only arose after #19402 .

For Firefox, https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/browsers/firefox.py#L132 add `acceptInsecureCerts` only if `certutil_binary` is not specified. In addition, https://github.com/web-platform-tests/wpt/blob/master/tools/wpt/run.py#L215 has the code to set `certutil_binary`, so `acceptInsecureCerts` is not actually added. (I wonder if the condition at https://github.com/web-platform-tests/wpt/blob/master/tools/wpt/run.py#L205 is intentional. During my test, `kwargs["ssl_type"]` has value of `None`, which doesn't equal to `"none"`, so the condition evaluates to true.)

**Comment by andreastt** [comment]
We don’t use the browser initialisation code referenced in https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/browsers/firefox.py#L132 for the wdspec type. In the wdspec case we rely entirely on what the WebDriver implementation does by default. `acceptInsecureCerts` is not the default in geckodriver for the wdspec tests.

**Review by andreastt — APPROVED** [review]
This change is fine though, it doesn’t affect geckodriver at all and solves the chromedriver case.

Do note that chromedriver _should not_ default to have `acceptInsecureCerts` on by default, so maybe this should be considered a chromedriver bug?

**Comment by Hexcles** [comment]
> Do note that chromedriver should not default to have acceptInsecureCerts on by default, so maybe this should be considered a chromedriver bug?

That was the case (https://crbug.com/chromedriver/3148), and it's been fixed, which is why I needed to change wptrunner to explicitly ignore cert errors in Chrome, but the change inadvertently affected wdspec tests.

And thanks for the explanation!

**Comment by Hexcles** [comment]
Taskcluster detected a slow test on Firefox:
https://tools.taskcluster.net/groups/P2tFfOfAQN-f6F0SU9bE0Q/tasks/JchWgygIQZyKjHb0TGWDCQ/runs/0/logs/public%2Flogs%2Flive.log#L16673

It's definitely unrelated to this change so I'm going to admin-merge it.

**Comment by andreastt** [comment]
Ah, thanks for the explanation!

**Pull request merged**